rust/crates/langsmith-tracing-client/src/client/run.rs

```diff
@@ -2,7 +2,7 @@ use serde::{Deserialize, Serialize};
 use serde_json::Value;
 
 // Map attachment ref to tuple of filename, optional bytes
-#[derive(Debug)]
+#[derive(Debug, Serialize, Deserialize)]
```

Comment:
I made all of these `Serialize, Deserialize` to make it easier to create them from JS.

This is a temporary hack to get something working quickly. We can figure out a more appropriate long-term solution once we have an end-to-end working implementation.